docs/ReleaseNotes.md

```diff
@@ -1,6 +1,45 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 1.6.1
+Acuminator 1.6.1 includes the diagnostics, code fixes, and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### New Diagnostics and Code Fixes
+In this version, the diagnostic and a code fix have been added for the following issue.
+
+| Code                            | Issue Description                                                                                           | Type    | Diagnostics | Code Fix    | 
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------- | ------- | ----------- | ----------- |
+| [PX1013](diagnostics/PX1013.md) | The action handler that initiates a background operation or is executed by a background operation must return `IEnumerable`. | Error | Available   | Available   |
+
+### Enhancements and Bug Fixes in the Code Map
+Acuminator 1.6.1 includes the following enhancements in the code map:
+
+ - In the text editor of Visual Studio, the symbol to which the code map navigates is now highlighted.
+ - The tooltip for a data view in the code map now keeps the BQL formatting of the source code.
+ - The namespace prefixes have been removed from the tooltips in the code map.
+ - The categories without child nodes are no longer displayed in the code map.
+ - The code map now displays the actions and methods of the base graph that have been overridden.
```

Comment:
@k-s-popova only methods could be overriden, I think actions are redundant here. Also it probably should be specified that "overidden" means override via `PXOverride` attribute. The overrides of virtual methods via standard C# override are not displayed in code map currently

Reply:
fixed

Reply:
ok